src/go/pt-galera-log-explainer/sed.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+/*
+type sed struct {
+	Paths []string `arg:"" name:"paths" help:"paths of the log to use"`
```

Comment:
Do we need this file?

Reply:
Seen with Sveta, we can keep
It should have been included in this PR but it will re-added in another one, there's an usage debate with that one